### PR TITLE
Documentation on using DIRC CLI via CVMFS

### PR DESCRIPTION
**Pull request by ginomarchetti**
Information added on the access to DIRAC client via CVMFS to document DIRAC-Notebook integration. 

# Summary
Changes in : 

- `content/en/users/compute/orchestration/workload-manager/_index.md`: added CVMFS information, changed the header level for **Technical Support** paragraph
- `content/en/users/dev-env/notebooks/integration/_index.md`: added CVMFS information
---

**Related issue :**

Answers JIRA ticket https://jira.egi.eu/browse/ACETA-142

### CODE REVIEW

File: content/en/users/dev-env/notebooks/integration/_index.md [hunk 1]
```diff
@@ -108,6 +108,15 @@ print(execution.status)
 Note that inputs that point to a URL should be specified using the
 `ComplextDataInput` class as shown above.
 
+## EGI-WMS (DIRAC)
+
+If you are using a Notebooks instance integrated with D4Science, you may access the DITAC client via CVMFS with the following two command lines:
```
Comment (sebastian-luna-valero) [inline content/en/users/dev-env/notebooks/integration/_index.md:113]:
```suggestion
If you are using a Notebooks instance integrated with D4Science, you may access the DIRAC client via CVMFS with the following two command lines:
```

**Review by sebastian-luna-valero — APPROVED** [review]
Thanks, very happy to see this!

**Review by gwarf — CHANGES_REQUESTED** [review]
Thanks!
Could you also add `$` in front of all shell commands, in the other already existing code blocks?

**Comment by ginomarchetti** [comment]
> Thanks! Could you also add `$` in front of all shell commands, in the other already existing code blocks?
> 
Done

File: content/en/users/dev-env/notebooks/integration/_index.md [hunk 2]
```diff
@@ -108,6 +108,15 @@ print(execution.status)
 Note that inputs that point to a URL should be specified using the
 `ComplextDataInput` class as shown above.
 
+## EGI-WMS (DIRAC)
```
Comment (enolfc) [inline content/en/users/dev-env/notebooks/integration/_index.md:111]:
```suggestion
## EGI Workload Manager (DIRAC)
```

Could you also move this section to before the D4Science one?
Also add a reference to the DIRAC section to the docs

**Review by enolfc — CHANGES_REQUESTED** [review]
Thanks @ginomarchetti! Just some minor changes needed.

File: content/en/users/dev-env/notebooks/integration/_index.md [hunk 3]
```diff
@@ -108,6 +108,15 @@ print(execution.status)
...
+
+If you are using a Notebooks instance integrated with EGI Workload Manager, you may access the DIRAC client via CVMFS with the following two command lines:
```
Comment (gwarf) [inline content/en/users/dev-env/notebooks/integration/_index.md:113]:
```suggestion
If you are using a Notebooks instance integrated with EGI Workload Manager,
you may access the DIRAC client via CVMFS with the following two command lines:
```

**Review by gwarf — APPROVED** [review]

File: content/en/users/dev-env/notebooks/integration/_index.md (hunk 2 above)
Comment (ginomarchetti) [reply content/en/users/dev-env/notebooks/integration/_index.md]: Added all suggestions.

Comment (enolfc) [reply content/en/users/dev-env/notebooks/integration/_index.md]: Could you also tackle the other comments?
- Move this section to before the D4Science one?
- Add a reference to the DIRAC section to the docs

Thanks!

**Review by enolfc — APPROVED** [review]

**Pull request merged**